config/environments/test.rb

```diff
@@ -44,16 +44,19 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
-  # todo
+  # todo no need until we add disallowed deprecations
   # Raise exceptions for disallowed deprecations.
   # config.active_support.disallowed_deprecation = :raise
 
+  # todo no need until we add disallowed deprecations
   # Tell Active Support which deprecation messages to disallow.
   # config.active_support.disallowed_deprecation_warnings = []
 
+  # todo could enable?
   # Raises error for missing translations.
   # config.i18n.raise_on_missing_translations = true
```

Comment:
yes lets enable

Reply:
I think this is on globally in `application.rb`?

Reply:
ah yes, can remove